leshoya.rb

```diff
@@ -0,0 +1,176 @@
+# PRODUCT HAS A NAME, DIAMETER, MASS, MOONS, GRAVITY, AND DISTANCE FROM THE SUN VALUES
+class Planet #always camel case
+
+  attr_reader :name, :diameter, :mass, :moons, :gravity, :distance_from_the_sun
+  attr_writer :name, :diameter, :mass, :moons, :gravity, :distance_from_the_sun
+
+
+  def initialize (name, diameter, mass, moons,gravity, distance_from_the_sun)
+
+    @name=name
```

Comment:
Super nit-picky style comment, typically `=` signs have spaces on both sides of them.
```rb
@name = name
```